pom.xml

```diff
@@ -60,6 +60,17 @@
 			<artifactId>mysql</artifactId>
 			<scope>test</scope>
 		</dependency>
+<!--		<dependency>-->
+<!--			<groupId>org.liquibase</groupId>-->
+<!--			<artifactId>liquibase-core</artifactId>-->
+<!--			<version>4.27.0</version>-->
+<!--		</dependency>-->
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>1.5.5.Final</version>
```

Comment:
ai putea scoate intr-o variabila la properties versinuile, asa cum ai la lombok

Reply:
am cautat pe net si am facut pentru mapstruct, dar la lombok nu vad unde am asta in properties, ne uitam